### PR TITLE
[BUG] fix the download and upload path

### PR DESCRIPTION
**Pull request by jinmingteo**
Edit download and upload directory in the example

### CODE REVIEW

File: _posts/2020-07-13-ml-dl-model-training-on-aiplatform-part1.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ s3 = boto3.resource('s3',
 
 def s3_download_file(localfile,bucket,s3path):
     print("S3 Download s3://"+bucket+"/" + s3path + " to " + localfile )
-    s3.Bucket(bucket).download_file(s3path,localfile)
+    s3.Bucket(bucket).download_file(os.path.join(s3path, localfile), localfile)
```
Comment (jax79sg) [inline _posts/2020-07-13-ml-dl-model-training-on-aiplatform-part1.md:91]: Hi Jinming, sorry, looking at this again. I don't think the change should be like this. I do however agree that the original code is ambiguous as well. So basically if we change the code to this, we are assuming we are download the S3 file and saving it as the same filename. However, people can choose to save as a different path and filename. So i recomend changing names only instead.

s3path change to s3pathandfilename
localfile change to localpathandfilename

**Review by jax79sg — CHANGES_REQUESTED** [review]
Hi Jinming, sorry, looking at this again. I don't think the change should be like this. I do however agree that the original code is ambiguous as well. So basically if we change the code to this, we are assuming we are download the S3 file and saving it as the same filename. However, people can choose to save as a different path and filename. So i recomend changing names only instead.

`s3path` change to `s3pathandfilename`
`localfile` change to `localpathandfilename`

File: _posts/2020-07-13-ml-dl-model-training-on-aiplatform-part1.md (hunk 1 above)
Comment (jinmingteo) [reply _posts/2020-07-13-ml-dl-model-training-on-aiplatform-part1.md]: Hi Kah Siong,

agreed with your changes. I guess the part i was confused about was the subdirectory that we needed in order to reach the s3 files. (for example, digitalhub/**cat_dog**/kagglecatsanddogs_3367a.zip).

Hopefully this commit will clear the confusion. Maybe we will have to edit this page too
https://raw.githubusercontent.com/jax79sg/kubejob/master/single-train/image_classification_single.py

**Review by jax79sg — APPROVED** [review]
Thanks a lot. @jinmingteo

**Pull request merged**